app/controllers/api_resource_comments_controller.rb

```diff
@@ -4,4 +4,46 @@ def show
     comments = Resource.find(resource_id)&.resource_comments
     render json: comments, only: %i[id content resource_id user_id]
   end
+
+  def create
+    content = params['content']
+    resource_id = params['resource_id']
+    comment = ResourceComment.new(content:, user: current_user, resource_id:)
+
+    if comment.save
+      render_successful_response(comment)
```

Comment:
I don't know if it is necessary to create a custom method. Why not use render json?